vendor/k8s.io/kubernetes/pkg/kubelet/util/pluginwatcher/plugin_watcher_test.go

```diff
@@ -212,6 +212,7 @@ func TestExamplePlugin(t *testing.T) {
 func waitForPluginRegistrationStatus(t *testing.T, statusCh chan registerapi.RegistrationStatus) bool {
 	select {
 	case status := <-statusCh:
+		time.Sleep(time.Millisecond)
```

Comment:
This is still flake prone at load. If a sleep is needed, sleep for longer.

Reply:
increased to 1 sec, i think it should be enough.